lib/ret_web/channels/hub_channel.ex

```diff
@@ -234,8 +234,12 @@ defmodule RetWeb.HubChannel do
 
   def handle_in("events:begin_recording", _payload, socket), do: socket |> set_presence_flag(:recording, true)
   def handle_in("events:end_recording", _payload, socket), do: socket |> set_presence_flag(:recording, false)
+  def handle_in("events:raise_hand", _payload, socket), do: socket |> set_presence_flag(:handRaised, true)
+  def handle_in("events:lower_hand", _payload, socket), do: socket |> set_presence_flag(:handRaised, false)
```

Comment:
I would suggest using snake_case for tokens, `:hand_raised` instead of `:handRaised`, since that's the style we use consistently in elixir code.

Reply:
Updated.

Reply:
:+1: Just a reminder that you'll have to update the corresponding code in the client.